src/main/java/muffin/Muffin.java

```diff
@@ -0,0 +1,178 @@
+package muffin;
+
+import java.util.Scanner;
+
+/**
+ * The Muffin class represents the main application logic for a todo list.
+ */
+public class Muffin {
+
+    /**
+     * The file path to the task list file.
+     */
+    String filePath = "../taskList.txt";
+
+    /**
+     * Scanner object to capture user input.
+     */
+    Scanner sc = new Scanner(System.in);
+
+    /**
+     * An instance of FileProcessor to handle reading and writing tasks to a file.
+     */
+    FileProcessor fp = new FileProcessor();
```

Comment:
It may be more suitable for this variable to have a more descriptive name, as it is used throughout the whole program. Maybe a better name can be `fileProcessor`